templates/CRM/Activity/Form/Selector.tpl

```diff
@@ -102,7 +102,14 @@
 
     <td>{$row.activity_status}</td>
 
-    <td>{$row.action|replace:'xx':$row.id}</td>
+    <td>
+      {if (!empty($row.id))}
+        {$row.action|replace:'xx':$row.id}
```

Comment:
I had left this one out of a previous PR because I don't think this replacement ever actually happens, just hadn't got around to confirming it. Not a blocker just I might end up forgetting about it if it never displays again.

Reply:
Yeah I wondered about that